chain/paseo/defaults.go

```diff
@@ -5,15 +5,12 @@ package paseo
 
 import (
 	cfg "github.com/ChainSafe/gossamer/config"
+	"github.com/adrg/xdg"
 )
 
 var (
-	// defaultName Default node name
-	defaultName = "Paseo"
-	// defaultID Default chain ID
-	defaultID = "paseo"
```

Comment:
why remove ID and Name?

Reply:
Because this is set through the config/cli interface. Name and ID should not be overridden by the chain specification. See #3685